Rakefile

```diff
@@ -17,7 +17,7 @@ task :default => [:test]
 
 desc 'Run RSpec'
 RSpec::Core::RakeTask.new(:test) do |t|
-  t.pattern = 'spec/{unit}/**/*.rb'
+  t.pattern = 'spec/{classes,defines,unit,functions,hosts,integration}/**/*_spec.rb'
```

Comment:
@ferventcoder understood. After looking a bit closer and discussing with @ThoughtCrhyme and @glennsarti I think we can remove this `rake::test` task entirely and defer to `rake::spec`. This will require an associated PR to JJB to ensure our `Spec Tests` phase of the pipeline runs the right thing. Will link when that is up.